src/components/resources.css

```diff
@@ -0,0 +1,122 @@
+.main-content {
+    margin: 4em auto 0;
+    max-width: 800px;
+}
+
+.palette-grid {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-template-rows: auto 300px auto 240px auto repeat(2, 200px);
+    grid-gap: 20px;
```

Comment:
We should be using EM's or REM's instead of pixel units for almost all attributes.

Reply:
Done

Reply:
There's still a lot of PX units used throughout the styling. Should be able to do a simple conversion then find and replace all of the various 3px, 4px, 12px, 14px, etc. values

Reply:
I've gone thru and eliminated them all. everything should be in em's now.